models/user.js

```diff
@@ -1,23 +1,19 @@
 var mongoose = require('mongoose');
 var Schema = mongoose.Schema;
 
-var valuesSchema = mongoose.Schema({
-  interests: [String]
-});
-
-var mechanicsSchema = mongoose.Schema({
-  cleanliness: Number,
-  noise: Number,
-  pets: Boolean,
-  sleep: String
-});
-
 var userSchema = mongoose.Schema({
   name: String,
   email: String,
   about: String,
-  values: valuesSchema,
-  mechanics: mechanicsSchema 
+  values: {
+    interests: [String]
+  },
+  mechanics: {
+    cleanliness: Number,
+    noise: Number,
+    pets: Boolean,
+    sleep: String
+  }
```

Comment:
Cool I agree. And as for PostGres, I know that it is more stable than MongoDB in terms of safe writing and memory usage. It would make sense to transition earlier than later. Everything looks great.